src/components/Square.js

```diff
@@ -1,19 +1,15 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import './Square.css'
+import './Square.css';
 
 const Square = (props) => {
   // For Wave 1 enable this 
   //  Component to alert a parent 
   //  component when it's clicked on.
 
-  return <button
-    className="square"
-  >
-    {props.value}
-  </button>
-}
+  return <button className='square' onClick={() => props.onClickCallback(props.id)}>{props.value}</button>;
+};
```

Comment:
We can use object destructuring to reference the props data by their key name instead of using dot notation:

```
const Square = ( {value, id, onClickCallback, key} ) => { 
    return <button className='square' onClick={() => onClickCallback(id)}>{value}</button>;
};
}
```

Also, good work in using an anonymous callback function for the button's `onClick`.